egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -83,20 +83,51 @@ def AddOutputNode(config_lines, input, label_delay=None):
 def AddFinalLayer(config_lines, input, output_dim, ng_affine_options = "", label_delay=None, include_softmax = "true"):
```

Comment:
@danpovey The number of lines of python code in kaldi is increasing due to the nnet3 config generators. Do you think we should start enforcing consistency of style using [Google's python style guide](https://google.github.io/styleguide/pyguide.html). There also exists corresponding checking tools ([link](https://github.com/google/styleguide/))